projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -358,56 +316,130 @@ public static void saveTrips(Path csvPath, String path) {
+        trips.add(
+            trip.setRouteId(Integer.parseInt(record.get("route_id")))
+                .setServiceId(Integer.parseInt(record.get("service_id")))
+                .setTripId(record.get("trip_id"))
+                .setTripHeadsign(record.get("trip_headsign"))
+                .setDirectionId(Integer.parseInt(record.get("direction_id")))
+                .setShapeId(record.get("shape_id"))
+                .build());
+      }
+      return trips;
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public static List<String> getStopsName() {
+    return getStops(stopsAbsPath).stream().map(Stop::getStopName).collect(Collectors.toList());
+  }
+
+  public static void saveCalendar() {
+    try (FileOutputStream output = new FileOutputStream(calendarProtoAbsPath)) {
+      for (Protos.Calendar calendar : getCalendar(calendarAbsPath)) {
+        calendar.writeDelimitedTo(output);
+      }
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public static void saveRoutes() {
+    try (FileOutputStream output = new FileOutputStream(routesProtoAbsPath)) {
+      for (Route route : getRoutes(routesAbsPath)) {
+        route.writeDelimitedTo(output);
+      }
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public static void saveStopTime() {
+    try (FileOutputStream output = new FileOutputStream(stopTimeProtoAbsPath)) {
+      for (StopTime stopTime : getStopTimes(stopTimeAbsPath)) {
+        stopTime.writeDelimitedTo(output);
+      }
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public static void saveStops() {
+    try (FileOutputStream output = new FileOutputStream(stopsProtoAbsPath)) {
+      for (Stop stop : getStops(stopsAbsPath)) {
+        stop.writeDelimitedTo(output);
+      }
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  public static void saveTranslations() {
+    try (FileOutputStream output = new FileOutputStream(translationsProtoAbsPath)) {
+      for (Translation translation : getTranslations(translationsAbsPath)) {
+        translation.writeDelimitedTo(output);
       }
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
   }
 
+  public static void saveTrips() {
+    try (FileOutputStream output = new FileOutputStream(tripsAbsProtoAbsPath)) {
+      for (Trip trip : getTrips(tripsAbsPath)) {
+        trip.writeDelimitedTo(output);
+      }
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  // Paths to the CSVs files
+  static Path calendarAbsPath;
+  static Path routesAbsPath;
+  static Path stopTimeAbsPath;
+  static Path stopsAbsPath;
+  static Path translationsAbsPath;
+  static Path tripsAbsPath;
+
+  // Paths to output files
+  static String calendarProtoAbsPath;
+  static String routesProtoAbsPath;
+  static String stopTimeProtoAbsPath;
```

Comment:
It's better to be consistent - all plural or all singular:
`stopTimeProtoAbsPath>stopTimesProtoAbsPath`

Reply:
Correct! Fixed also in other places to keep all as plural